godel.rosinstall

```diff
@@ -1,10 +1,15 @@
 - git: {local-name: descartes, uri: 'https://github.com/ros-industrial-consortium/descartes.git',
-    version: indigo-devel}
+    version: kinetic-devel}
 - git: {local-name: godel_openvoronoi, uri: 'https://github.com/ros-industrial-consortium/godel_openvoronoi.git',
     version: hydro-devel}
 - git: {local-name: godel, uri: 'https://github.com/ros-industrial-consortium/godel.git',
-    version: indigo-devel}
+    version: kinetic-devel}
 - git: {local-name: motoman, uri: 'https://github.com/ros-industrial/motoman.git',
     version: indigo-devel}
-- git: {local-name: abb, uri: 'https://github.com/ros-industrial/abb.git',
-    version: indigo-devel}
+
+# Custom abb until Kinetic release happens
+- git: {local-name: abb, uri: 'https://github.com/Jmeyer1292/abb.git',
+    version: kinetic-devel}
+
+- git: {local-name: industrial_core, uri: 'https://github.com/ros-industrial/industrial_core.git',
+    version: kinetic-devel}
```

Comment:
Sure. I just tried to update and install ros-kinetic-industrial-core and it's not there yet. I will update it when these are released.

Do you have any good resources on the differences between all of the different ros dependency tools? `rosdep` vs `rosinstall` vs `wstool`? I'm slowly starting to care about being able to on-board others easily and would like to get this sorted out.